server/src/api/modules/groups/services/EditGroupService.ts

```diff
@@ -175,19 +183,56 @@ async function executeUpdate(params: EditGroupParams, updatedGroupFields: Prisma
   const joinedEvents: MemberJoinedEvent[] = [];
   const changedRoleEvents: MemberRoleChangeEvent[] = [];
 
+  const nameChanges = (await prisma.nameChange.findMany({
+    where: {
+      OR: missingPlayers.map(p => {
+        const startsWith = p.username;
+        return { newName: { startsWith, mode: 'insensitive' } };
```

Comment:
why startsWith and not just regular equals?

if there's a name change from `Zezima` to `Jake` we wouldn't want that to interrupt `Jake 123`'s activity, right?